src/Message/Message.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { toast, ToastContainer } from "react-toastify";
+import Styles from "./message.style";
+import { Icon } from "../index";
+
+function MessageContent({ text, icon, color, size }) {
+  return (
+    <Styles>
+      <div className="text-content">
+        <Icon name={icon} style={{ color, fontSize: size }} />
+        &nbsp;
+        {text}
+      </div>
+    </Styles>
+  );
+}
+
+MessageContent.propTypes = {
+  text: PropTypes.string.isRequired,
+  icon: PropTypes.string.isRequired,
+  color: PropTypes.string,
+  size: PropTypes.number
+};
+
+MessageContent.defaultProps = {
+  color: "#04d4bf",
+  size: 22
+};
+
+export function MessageContainer() {
+  return (
+    <Styles>
+      <ToastContainer />
+    </Styles>
+  );
+}
+
+export const message = {
+  ...toast,
+  info(text, options = {}) {
```

Comment:
can you explain, maybe with a story how this `options` can be used?

Reply:
There's a documentation story that explains it